pypesto/objective/amici_objective.py

```diff
@@ -2,13 +2,18 @@
 import copy
 import logging
 import numbers
+from typing import Dict, List, Tuple, Union
+from collections import OrderedDict
+
 from .objective import Objective
 from .constants import MODE_FUN, MODE_RES, FVAL, GRAD, HESS, RES, SRES, RDATAS
+from .options import ObjectiveOptions
 
 try:
     import amici
+    import amici.petab_objective
 except ImportError:
```

Comment:
Not sure if this has any effect. If import fails, the typehints below will cause trouble. Would have to quote all amici types I think.

Reply:
right. done, also for petab.